vllm/platforms/xpu.py

```diff
@@ -0,0 +1,49 @@
+import os
+from typing import Tuple
+
+import torch
+
+from vllm.utils import print_warning_once
+
+from .interface import Platform, PlatformEnum
+
+
+def device_id_to_physical_device_id(device_id: int) -> int:
+    if "XPU_VISIBLE_DEVICES " in os.environ:
+        device_ids = os.environ["XPU_VISIBLE_DEVICES "].split(",")
```

Comment:
Is the extra whitespace here intended?

Reply:
This whitespace is added after running `bash format.sh`, i will figure out which rule it follows.